src/components/SelectionList/BaseSelectionList.js

```diff
@@ -290,121 +301,114 @@ function BaseSelectionList({
     useKeyboardShortcut(CONST.KEYBOARD_SHORTCUTS.ENTER, selectFocusedOption, {
         captureOnInputs: true,
         shouldBubble: () => !flattenedSections.allOptions[focusedIndex],
-        isActive: !activeElement && isFocused,
+        isActive: !disableKeyboardShortcuts && !activeElement && isFocused,
     });
 
     /** Calls confirm action when pressing CTRL (CMD) + Enter */
     useKeyboardShortcut(CONST.KEYBOARD_SHORTCUTS.CTRL_ENTER, onConfirm, {
         captureOnInputs: true,
         shouldBubble: () => !flattenedSections.allOptions[focusedIndex],
-        isActive: Boolean(onConfirm) && isFocused,
+        isActive: !disableKeyboardShortcuts && Boolean(onConfirm) && isFocused,
     });
 
     return (
-        <ArrowKeyFocusManager
```

Comment:
This huge block of changes is actually just removing `<ArrowKeyFocusManager />` that was wrapping everything